conda/core/install.py

```diff
@@ -185,21 +188,24 @@ def _make_link_operations(self, requested_link_type):
         site_packages_dir = get_site_packages_dir(self.prefix)
         bin_dir = get_bin_directory_short_path()
 
-        def make_link_operation(source_short_path):
+        def make_link_operation(source_path_info):
             # no side effects in this method!
 
             # first part, same as parent class
-            if source_short_path in package_info.has_prefix_files:
+            if getattr(source_path_info, "prefix_placeholder", None) is not None:
```

Comment:
Here you could just add `required=False, nullable=True` to the `prefix_placeholder` field so you don't have to use `getattr`.